### PR TITLE
DHFPROD-5614: Confusing Placeholder Text in Mapping Modal

### PR DESCRIPTION
**Pull request by briantang**
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

File: marklogic-data-hub-central/ui/src/components/entities/mapping/create-edit-mapping-dialog/create-edit-mapping-dialog.test.tsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ describe('Create/Edit Mapping Step artifact component', () => {
     expect(getByPlaceholderText('Enter description')).toBeInTheDocument();
     expect(getByLabelText('Collection')).toBeInTheDocument();
     expect(getByLabelText('Query')).toBeInTheDocument();
-    expect(getByPlaceholderText('Please select')).toBeInTheDocument();
+    expect(getByPlaceholderText('Enter Collection')).toBeInTheDocument();
```
Comment (rjrudin) [inline marklogic-data-hub-central/ui/src/components/entities/mapping/create-edit-mapping-dialog/create-edit-mapping-dialog.test.tsx:18]: I believe there's another JIRA for this where the decision was to do "Enter collection name". "Enter collection" is fine, but "C" needs to be "c".

Comment (briantang) [reply marklogic-data-hub-central/ui/src/components/entities/mapping/create-edit-mapping-dialog/create-edit-mapping-dialog.test.tsx:18]: @rjrudin I noticed there were some discrepancies with the casing on the relevant forms. There's 'Enter name' and 'Enter description' but for the Query input that is next to Collection, it uses 'Enter Source Query' so I chose to keep the casing consistent with that. Agree that 'Enter collection name' may be better. Maybe a question for UX? @jbelonoj

Comment (jbelonoj) [reply marklogic-data-hub-central/ui/src/components/entities/mapping/create-edit-mapping-dialog/create-edit-mapping-dialog.test.tsx:18]: Thanks for bringing this up, @briantang! Yes, for ALL placeholder text we agreed on this format: "Enter collection name". Also, please note that we do not start with "Please", we start with "Enter" or "Select" in most cases. First word starts with capital letter, all following words start with lowercase letters.

Comment (briantang) [reply marklogic-data-hub-central/ui/src/components/entities/mapping/create-edit-mapping-dialog/create-edit-mapping-dialog.test.tsx:18]: Ok @jbelonoj thanks for clearing that up. Will also fix the casing in Query from 'Enter Source Query' to 'Enter source query' as part of this PR.

**Comment by briantang** [comment]
@rjrudin thanks for the catch. Relevant adjustments have been made.

**Review by timur-isangulov — DISMISSED** [review]

**Review by wooldridge — APPROVED** [review]

**Review by brucean52 — APPROVED** [review]

**Review by timur-isangulov — APPROVED** [review]

**Pull request closed without merging**